django/test/client.py

```diff
@@ -674,9 +674,9 @@ def _login(self, user, backend=None):
 
     def logout(self):
         """
-        Removes the authenticated user's cookies and session object.
+        Remove the authenticated user's cookies and session object.
```

Comment:
"Log out the user by removing the cookies and session object."
(remove second sentence)